modules/tinymce/src/core/main/ts/content/GetContentImpl.ts

```diff
@@ -22,9 +22,17 @@ const getContentFromBody = (editor: Editor, args: GetContentArgs, body: HTMLElem
   if (args.format === 'raw') {
     content = Tools.trim(TrimHtml.trimExternal(editor.serializer, body.innerHTML));
   } else if (args.format === 'text') {
-    // return empty string for text format when editor is empty to avoid bogus elements being returned in content
-    content = Zwsp.trim(body.innerText);
-    content = content === '\n' ? '' : content;
+    const doc = editor.getDoc();
+    content = body.innerHTML.replace(/<br data-mce-bogus="1">/g, Unicode.zeroWidth);
+    const parseBody = doc.createElement('body');
```

Comment:
I'm still pretty concerned about us having 2 body elements, as it's not valid HTML. So while it might work now, it might break easily in the future. I really think we need to try and find a different solution here.

Reply:
I agree, but if we use a div we will get different output on different browsers. If you have a workaround or better solution I'd be delighted to hear it.

Reply:
I've spiked out a solution which I believe works, that doesn't need to append 2 body elements (or at least all the tests pass): https://github.com/tinymce/tinymce/compare/feature/TINY-8579...spike/TINY-8579. Take a look at that and see if you can see any issues.